conductor/blueprints/authorization/controllers.py

```diff
@@ -69,21 +72,9 @@ def __call__(self):
         return PUBLIC_KEY
 
 
-class Sample:
```

Comment:
I think class `Lib` here just a function:
```
def lib():
    url = 'http://localhost:8000'+url_for('.public-key') + resp = make_response(LIBJS)
    public_key = requests.get(url).text + resp.headers['Content-Type'] = 'text/javascript'
    token = request.values.get('jwt') + return resp
```
I supose @akariv has a reason to have callable classes here like design unification.

But I agree reasonable review point.